microsoft-edge/webview2/how-to/print.md

```diff
@@ -0,0 +1,744 @@
+---
+title: Printing from WebView2 apps
+description: How to print a webpage to a printer, PDF file, or PDF stream in a WebView2 app.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/12/2023
+---
+# Printing from WebView2 apps
+
+There are several different ways to print a web page in WebView2, which give you various levels of ease of implementation and customizing.
+
+| Method | Description |
+|---|---|
+| `ShowPrintUI` | Opens the WebView2 **Print Preview** dialog or the operating system's **Print** dialog. Easy to implement, minimal support for customizing. |
+| `Print` | Silently prints the current top-level document in the WebView2 using optional programmatically specified print settings to a printer. You can to build your own print preview dialog or print experience. |
```

Comment:
Nit unclear sentence: "you can to build your own..."  --> "you can use this to build your own..."